ElectronClient/gui/KeymapConfigScreen.tsx

```diff
@@ -0,0 +1,229 @@
+import * as React from 'react';
+const CommandService = require('lib/services/CommandService').default;
+const { DOMtoElectronAccelerator } = require('lib/KeymapUtils.js');
+const { themeStyle } = require('lib/theme');
+const { _ } = require('lib/locale.js');
+// const { bridge } = require('electron').remote.require('./bridge');
+// const Setting = require('lib/models/Setting');
+
+interface KeymapItem {
+	label: string,
+	accelerator: string,
+	command: string
+	isEditing: boolean,
+}
+
+// Temporary values
+const commandNames = [
+	'textCopy',
+	'textCut',
+	'textPaste',
+];
+const getAccelerator = (commandName: string) => {
+	switch (commandName) {
+	case 'textCopy':
+		return 'Ctrl+C';
+	case 'textCut':
+		return 'Ctrl+X';
+	case 'textPaste':
+		return 'Ctrl+V';
+	default:
+		throw new Error('Invalid command');
+	}
+};
+
+const initialKeymap = commandNames.map(commandName => {
+	return {
+		label: CommandService.instance().label(commandName),
+		accelerator: getAccelerator(commandName),
+		command: commandName,
+		isEditing: false,
+	};
+});
```

Comment:
That should be loaded inside the component, using useEffect for example. In general, only constants or functions can be defined outside the component. Any state-related variables should be defined inside the component (or as hooks).

Reply:
I've passed a state initialization function to useState(). It'll only be initialized once per lifetime.